Angelus/RosVista/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_cd_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/Angelus RosVista2 16x9 1920x1080.jpg" DESTINATION reactos/Web/Wallpaper FOR all)
```

Comment:
What did "WP" in "ROSWP" refer to, in the original name? Does this relate to "Vista"?

Reply:
Wallpaper?

Reply:
`VistaReactOS`.